site/content/extend/customization/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Customizing Mattermost"
+date: "2017-10-29T13:47:55+09:00"
+section: "extend"
+---
+
+Mattermost allows for a variety of customization options and variations, making it possible to create a more adequate experience depending on the needs of each deployment.
+
+There are a few limitations regarding [how the re-branding of Mattermost](https://www.mattermost.org/trademark-standards-of-use/) must be handled, as well as the fact that changes to the Enterprise Edition's source code isn't supported. However these can be overcome with the utilization of [Plugins](/extend/getting-started).
+  
+# Customizable Components
+  
+## Server (Team Edition only)
+The [Mattermost server](https://github.com/mattermost/mattermost-server)'s source code, written in Golang, may be customized to deliver additional functionalities or meet specific security requirements. 
```

Comment:
... "customized to deliver additional functionalities or **to** meet specific security requirements. " ...